slurm/prodHelper.py

```diff
@@ -108,7 +117,7 @@ def makeEvtGenDecayBc(self):
       dec = Decays(mass=p.mass, mixing_angle_square=1)
 
       decay_table = decay_table.format(
-                         Bc_br0=dec.Bc_to_uHNL.BR,
+                         Bc_br0=dec.Bc_to_uHNL.BR*1000,
                          #pfx='anti_' if not self.domajorana else '',
```

Comment:
I have a small question here: why are the respective BR multiplied by 1000?

Reply:
I did this because in some cases the BRs were so low that EvtGen would read them as 0.
This way it does not happen. 
At the end EvtGen normalise the BRs of a given particle to 1.

Reply:
Makes perfect sense, thanks a lot